### DISCUSSION TITLE
Is there an alternative for Linux or Windows?

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [google/santa#686] [Q&A]
Are there any alternatives to Santa for Linux or Windows?

**Comment by russellhancox** [comment]
I'm not aware of any open-source projects like Santa for Windows or Linux but there are several commercial offerings.
I know nothing about any of these other than they exist:

For Windows:

- Airlock 
- Carbon Black Enterprise Protection (formerly Bit9)
- Microsoft Defender Application Control for Windows

For Linux:

- Airlock

**Reply by the123blackjack** [reply]
Hi Would consider SRP (Software Restriction Policies) on windows is what santa is on Mac.

Easy to use front end gui and configurators

1) Hard Configurator
2) Simple Software Restriction Policy

**Comment by angelorobo** [comment]
In 2024, is there a better alternative for app blocking in Windows or Linux?

**Reply by proximity666** [reply]
There is definitely real good one, but unfortunately from Broadcom/CATechnologies. Control Minder (now Symantec Privileged Identity Manager). It has broader functionality than Santa and it is really rather easy to deploy and manage ....

**Comment by pmarkowsky** [comment]
For Linux you have a bunch of solutions built on [fanotify(7)](https://man7.org/linux/man-pages/man7/fanotify.7.html) e.g. [fapolicyd](https://github.com/linux-application-whitelisting/fapolicyd) 

> I'm not aware of any open-source projects like Santa for Windows or Linux but there are several commercial offerings. I know nothing about any of these other than they exist:
> 
>...
> For Linux:
> 
> * Airlock

For Linux most older / complete solutions are built on [fanotify(7)](https://man7.org/linux/man-pages/man7/fanotify.7.html) e.g. [fapolicyd](https://github.com/linux-application-whitelisting/fapolicyd). You can find documentation for it [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_blocking-and-allowing-applications-using-fapolicyd_security-hardening).

Newer solutions on Linux can make use of eBPF and the KRSI Linux Security Module. However I'm unaware of an allowlisting product using it today.
